azkaban-common/src/main/java/azkaban/utils/Utils.java

```diff
@@ -325,6 +327,18 @@ public static String formatDuration(long startTime, long endTime) {
     return days + "d " + hours + "h " + minutes + "m";
   }
 
+  public static void setCurrentTimeMillisFixed(long millis) {
```

Comment:
I realized that joda-time is being used in the project already so switched to use `DateTimeUtils` and got rid of the getters/setters.